scripts/plan.py

```diff
@@ -0,0 +1,70 @@
+from github import Github 
+from github import Auth
+import json
+import os
+import sys
+
+pull_request_id = int(os.getenv("CIRCLE_PULL_REQUEST").split("/")[-1])
+commit_hash = os.getenv("CIRCLE_SHA1")
+token = os.getenv("GITHUB_TOKEN")
+auth = Auth.Token(token)
+
+g = Github(auth=auth)
+
+org_name = "Cyber4All"
+org = g.get_organization(org_name)
+
+repo = g.get_repo("Cyber4All/Security-Injections")
+print(repo)
+
+pullrequests = repo.get_pulls(state='open', sort='updated', base='main')
+pull_request = repo.get_pull(pull_request_id)
+latest_pr = ''
+commits = pull_request.get_commits()
+
+# if pullrequests.totalCount > 0:
+#     # Get the most recent PR
+#     latest_pr = pullrequests[0]
+#     commits = latest_pr.get_commits()
+#     print(f"Latest PR: {latest_pr.title} (#{latest_pr.number})")
+# else:
+#     print("No open pull requests found.")
+#     sys.exit()
+
+absoluteFilePaths = []
+comment = "Modules: "
+
+for c in commits:
+    print(c)
+    files = c.files
+    for f in files:
+        # print("File: " + f.filename)
+        path = os.path.abspath(f.filename)
+        if "content/" in path and path not in absoluteFilePaths:
+            absoluteFilePaths.append(path)
+
+if not absoluteFilePaths: # if no commits have changes to modules, exit
```

Comment:
Does not check length of lists as well? Because an empty list is declared above.

Reply:
After looping through the changed files in the commits, if any of those changes were made to the modules then it would've been added to the list, before this check on if the list is empty.

Reply:
a little confused on this here. Wouldnt this condition pass every time if you dont check length?

Reply:
Looks like the correct way to check if an array is empty to me @Cwagne17 @johngonz0 and it seems to work as intended. I'll double check